### ISSUE TITLE
The new UI is terrible 

### ISSUE THREAD

**Issue by MKCOOL142** [NeoApplications/Neo-Store#326]
Idk why there was a need to go away from the clean material you ui of v0.9.4 and below but the last two updates had really bad navigation bar designs, I'd prefer the 0.9.4 design over any of them and think you should revert back to that and instead work on the animation of that navigation bar to make it more like the material you animation for navigation bars

**Comment by Fxsch** [comment]
Wouldn't it be possible to add an option for this?

**Comment by machiav3lli** [comment]
@Fxsch Nope, I see no reason, logic and have no energy to maintain two different UIs. As mentioned, a FOSS project can always be forked…
